tests/bluechi_test/test.py

```diff
@@ -82,14 +83,21 @@ def setup(self) -> Tuple[bool, Tuple[BluechiControllerContainer, Dict[str, Bluec
         return (success, (ctrl_container, node_container))
 
     def gather_logs(self, ctrl: BluechiControllerContainer, nodes: Dict[str, BluechiNodeContainer]):
-        print("Collecting logs from all container...")
+        print("Collecting logs from all containers...")
 
         if ctrl is not None:
             ctrl.gather_journal_logs(self.tmt_test_data_dir)
 
         for _, node in nodes.items():
             node.gather_journal_logs(self.tmt_test_data_dir)
 
+        print("Collecting logs from test executor...")
```

Comment:
In my point of view this `print` can be removed. If you want to keep it, I think it would be better to move it to the `self.gather_journal_logs()`.